data/json/bionics.json

```diff
@@ -145,6 +145,12 @@
         "description": "Your eyes have quick-reacting transition lenses installed over them.  They negate glare penalties, partially protect you from bright flashes, and protect your eyes when welding."
     },{
         "type": "bionic",
+        "id": "bio_blindfold",
+        "name": "Optical Dampers",
+        "toggled": true,
+        "description": "The lenses installed over your eyes can be tinted to block out incoming light."
+	},{
```

Comment:
This line contains tab instead of spaces. Please set up your text editor properly and don't use tabs.